benchmarks/benchmarks/api/bench_metis_partition.py

```diff
@@ -0,0 +1,23 @@
+import time
+import dgl
+import torch
+import numpy as np
+
+from .. import utils
+
+
+@utils.skip_if_gpu()
+@utils.benchmark('time', timeout=1200)
+@utils.parametrize('graph_name', ['reddit', 'ogbn-product'])
+@utils.parametrize('k', [2, 4, 8])
+def track_time(graph_name, k):
+    device = utils.get_bench_device()
+    data = utils.process_data(graph_name)
+    graph = data[0]
+
+    # timing
+    t0 = time.time()
+    gg = dgl.transform.metis_partition(graph, k)
+    t1 = time.time()
```

Comment:
dry run? How slow is this?

Reply:
Added dry run on pubmed. It takes about 3 minutes including data loading, and about 1 minute for the partition part. Changed to iterate 3 times